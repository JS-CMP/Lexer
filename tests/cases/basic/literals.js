let a = undefined;
let b = null;
let c = true;
let d = false;
let e = 0;
let f = 1;
let g = 1.1;
let h = 'a';
let i = '';
let v = "a";
let w = "";
