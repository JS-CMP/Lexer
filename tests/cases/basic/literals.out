#include "types/JsAny.hpp"
#include "class/Console/Console.hpp"

int main() {JS::Any a=JS::Any();JS::Any b=JS::Any(nullptr);JS::Any c=JS::Any(true);JS::Any d=JS::Any(false);JS::Any e=JS::Any(0);JS::Any f=JS::Any(1);JS::Any g=JS::Any(1.1);JS::Any h=JS::Any("a");JS::Any i=JS::Any("");JS::Any v=JS::Any("a");JS::Any w=JS::Any("");}
